examples/userapi/05_conditionaldimension.ipynb

```diff
@@ -0,0 +1,729 @@
+{
```

Comment:
how about renaming this file as 05_conditional_dimension.ipynb ?

Reply:
Yes, renamed.